reference/import_export/product-import.rst

```diff
@@ -303,6 +305,53 @@ The service uses the class ``Akeneo\Component\Localization\Localize\AttributeCon
 
     Read the cookbook to add your own localizer  :doc:`/cookbook/localization/index`
 
+Product Processor
+-----------------
+
+This element receives items one by one, fetches or creates the related product, updates and validates it.
```

Comment:
creates (or fetches if it already exists) the related product, updates it and validates it.